backend/de.metas.ui.web.base/src/main/java/de/metas/ui/web/window/datatypes/json/JSONLookupValuesList.java

```diff
@@ -194,14 +194,14 @@ public void putOtherProperty(final String name, final String jsonValue)
 	}
 
 	@JsonSetter
-	public JSONLookupValuesList setDefaultValue(final String defaultValue)
+	public JSONLookupValuesList setDefaultId(final String defaultId)
```

Comment:
why rename to `defaultId`?

Reply:
I renamed it after consulting with Teo. A String parameter called "defaultValue" might trick the dev into thinking that the expected unique identifier is the "Value" column, which is usually a unique String and is present in a lot of our tables. 
Instead, the expected default there is the ID, e.g. the AD_BoilerPlate_ID . I think this name is less confusing.